pkg/hints/hint_utils/ids_manager.go

```diff
@@ -217,6 +229,21 @@ func (ids *IdsManager) InsertStructField(name string, field_off uint, value *May
 	return vm.Segments.Memory.Insert(addr.AddUint(field_off), value)
 }
 
+// Inserts Uint256 value into an ids field (given the identifier is a Uint256)
+func (ids *IdsManager) InsertUint256(name string, val Uint256, vm *VirtualMachine) error {
+	err := ids.InsertStructField(name, 0, NewMaybeRelocatableFelt(val.Low), vm)
+	if err != nil {
+		return err
+	}
+	return ids.InsertStructField(name, 1, NewMaybeRelocatableFelt(val.High), vm)
+
+}
```

Comment:
Similar comment to GetUint256, use `GetAddr` to get the base address and then use `Memory.Insert` on addr and addr+1